anyway/load_discussions.py

```diff
@@ -1,39 +0,0 @@
-# -*- coding: utf-8 -*-
-import argparse
-from .models import DiscussionMarker
```

Comment:
Well, it stopped working when these imports were added because you can't import from `.` in a main script. But now it's fixed anyway.